assets/scenes/load_scene_example.scn.ron

```diff
@@ -13,7 +13,12 @@
             y: 0.0,
             z: 0.0
           ),
-          rotation: (0.0, 0.0, 0.0, 1.0),
+          rotation: (
+            x: 0.0,
+            y: 0.0,
+            z: 0.0,
+            w: 0.0,
```

Comment:
shouldn't `w` be 1.0 to make it an identity quaternion like in old code?

Reply:
Thanks for catching that!